### ISSUE TITLE
MixedDatasource: Support streaming from many different data sources 

### ISSUE THREAD

**Issue by faxm0dem** [grafana/grafana#28981]
Using master 212bf7a4f6 and developing a streaming datasource, I stumbled upon this issue:

In the panel's query editor, when selecting my streaming datasource globally for all queries, everything works fine : data points appear on the graph as they are received. However, when selecting `-- Mixed--` and then the right DS in the query breaks the panel. No new data is displayed and the progress icon on the top right keeps rolling.

**Comment by faxm0dem** [comment]
I noticed you added the label `awaiting-more-evidence`, does this mean you expect anything from me?

**Comment by yayitazale** [comment]
Same here. I will like to mix redis with an influx datasource query, to load historical data from influx at the dashboard opening and then, keep filling the graph with redis real-time data.

**Comment by faxm0dem** [comment]
Hi, ist there anything we can do here ?
I just noticed the `prio/awaiting-more-evidence` flag is set. Do you need more data ?

thanks

**Comment by faxm0dem** [comment]
yeah well, we still have the issue and would very much like a fix, thank you github-cleanup-bot

**Comment by pedroapero** [comment]
> It will be closed in 30 days if no further activity occurs. Please feel free to leave a comment if you believe the issue is still relevant. 

Hi bot, yes this is a longstanding wish (currently using Promxy for this purpose).

**Comment by faxm0dem** [comment]
It really seems this has fallen into a black hole of Grafanalab's issue management procedure

**Comment by devthatcher0** [comment]
More people desire this feature: [https://github.com/grafana/grafana/issues/113806](https://github.com/grafana/grafana/issues/113806)

**Comment by faxm0dem** [comment]
@devthatcher0 thanks for linking both issues. However your urls don't point to the right place

**Comment by devthatcher0** [comment]
sorry just fixed, first time commenting on an issue
